packages/rsk3-account/tests/models/account.test.js

```diff
@@ -523,20 +443,8 @@ describe('AccountTest', () => {
 
         expect(() => {
             Account.fromPrivateKey(mockKey).toV3Keystore('password', options);
-        }).toThrow('Unsupported cipher');
+        }).toThrow('options.n should be number and has value of 2048, 4096, 8192 or 16384');
 
         expect(fromPrivate).toHaveBeenCalledWith(mockKey);
-
-        expect(randomBytes).toHaveBeenNthCalledWith(1, 32);
-
-        expect(randomBytes).toHaveBeenNthCalledWith(2, 16);
-
-        expect(pbkdf2Sync).toHaveBeenCalledWith(Buffer.from('password'), Buffer.from('random'), 262144, 32, 'sha256');
-
-        expect(createCipheriv).toHaveBeenCalledWith(
-            'aes-128-ctr',
-            Buffer.from('0000000000000000').slice(0, 16),
-            Buffer.from('random')
-        );
```

Comment:
This is a failed test case, randomBytes() is not called so the rest of toHaveBeenNthCalledWith() checks are not needed.